clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -4699,6 +4699,55 @@ TEST_F(FormatTestComments, SplitCommentIntroducers) {
                    getLLVMStyleWithColumns(10)));
 }
 
+TEST_F(FormatTestComments, LineCommentsOnStartOfFunctionCall) {
+  auto Style = getLLVMStyle();
+
+  EXPECT_TRUE(Style.Cpp11BracedListStyle);
+
+  verifyFormat("T foo( // Comment\n"
+               "    arg);",
+               Style);
+
+  verifyFormat("T bar{ // Comment\n"
+               "    arg};",
+               Style);
+
+  verifyFormat("T baz({ // Comment\n"
+               "        arg});",
```

Comment:
Why isn't this  continuation indent like others?

Reply:
Fixed.